sources/packages/backend/apps/api/src/route-controllers/institution/models/institution.dto.ts

```diff
@@ -132,6 +132,7 @@ export class InstitutionDetailAPIOutDTO extends InstitutionProfileAPIOutDTO {
   legalOperatingName: string;
   institutionTypeName?: string;
   isBCPrivate?: boolean;
+  isBCPublic?: boolean;
```

Comment:
Is there a reason for this one to be nullable?
If it is just following the `isBCPrivate` maybe both can be not nullable.